packages/web-config-server/src/apiV1/organisationUnit.js

```diff
@@ -12,12 +12,46 @@ const translateDescendantForFrontEnd = (descendant, childIdToParentId, entityIdT
   parent: entityIdToCode[childIdToParentId[descendant.id]],
 });
 
+async function fetchEntitiesWithProjectAccess(req, entities, userGroups) {
```

Comment:
Is this exactly the same as the function in the `project` endpoint handler? If so, can we make it a util?